yarn/src/main/scala/org/apache/spark/launcher/YarnCommandBuilderUtils.scala

```diff
@@ -31,11 +30,7 @@ private[spark] object YarnCommandBuilderUtils {
   }
 
   def findJarsDir(sparkHome: String): String = {
-    val scalaVer = Properties.versionNumberString
-      .split("\\.")
-      .take(2)
-      .mkString(".")
-    CommandBuilderUtils.findJarsDir(sparkHome, scalaVer, true)
+    CommandBuilderUtils.findJarsDir(sparkHome, true)
```

Comment:
Can we just get rid of this class if it just calls `CommandBuilderUtils`? I feel like I might be forgetting a reason it exists.